### ISSUE TITLE
Temporal data split

### ISSUE THREAD

**Issue by vshlepov** [asapdiscovery/asap-polaris-blind-challenge-examples#3]
Hi guys! The ADMET page mentions that "We used a temporal split". Is training data arranged chronologically or in some other fashion too, or it's just randomly shuffled? It's good to know that to mimic the test set and avoid unnecessary (if any) leakage.

Best,

Victor

**Comment by hmacdope** [comment]
Hi @vshlepov, the test data is arranged first to minimise leakage and then chronologically such that newest data is in the test set if possible. This mimics the lead-progression of a series as much as possible. Think of it as being a "time biased" split though rather than a true time-wise split due to the leakage constraints.

**Comment by vshlepov** [comment]
Thanks!
